skglm/penalties/__init__.py

```diff
@@ -1,6 +1,6 @@
 from .base import BasePenalty
 from .separable import (
-    L1_plus_L2, L0_5, L1, L2_3, MCPenalty, SCAD, WeightedL1, IndicatorBox
+    L1_plus_L2, L0_5, L1, L2_3, MCPenalty, SCAD, WeightedL1, IndicatorBox, SLOPE
```

Comment:
deserves discussion: SLOPE is not separable. Where do we put it ?

Reply:
What does it imply for the user? If we want to be rigorous, maybe we should put it in `experimental`.

Reply:
Whether in the experimental folder or in a non-separable.py file in the penalties folder.

Reply:
I agree with @Klopfe, we better add a new module ``non_seperable.py``. We've recently added FISTA solver which we intend to use for non-separable problems.